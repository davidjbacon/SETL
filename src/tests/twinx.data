"How extraordinary," he exclaimed.  "Was it really Exxon?"
