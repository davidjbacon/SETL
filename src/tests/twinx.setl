GETA('',s);
print('x'+/[IF c = 'x' THEN 'xx' ELSE c END: c IN s]+'x');
