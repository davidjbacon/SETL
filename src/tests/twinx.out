x"How exxtraordinary," he exxclaimed.  "Was it really Exxxxon?"x
